modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/upload/DDMFormUploadValidator.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.form.web.internal.upload;
+
+import com.liferay.document.library.kernel.exception.FileSizeException;
+import com.liferay.dynamic.data.mapping.form.web.internal.configuration.DDMFormWebConfiguration;
+import com.liferay.dynamic.data.mapping.form.web.internal.configuration.activator.DDMFormWebConfigurationActivator;
+import com.liferay.petra.string.StringBundler;
+
+import java.io.File;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.osgi.service.component.annotations.ReferencePolicyOption;
+
+/**
+ * @author Carolina Barbosa
+ */
+@Component(service = DDMFormUploadValidator.class)
+public class DDMFormUploadValidator {
+
+	public long getGuestUploadMaximumFileSize() {
+		DDMFormWebConfiguration ddmFormWebConfiguration =
+			_ddmFormWebConfigurationActivator.getDDMFormWebConfiguration();
+
+		return ddmFormWebConfiguration.guestUploadMaximumFileSize() *
+			_FILE_LENGTH_MB;
+	}
+
+	public void validateFileSize(File file, String fileName)
+		throws FileSizeException {
+
+		if (file == null) {
+			throw new FileSizeException("File is null for " + fileName);
```

Comment:
@carolmariaabb, is there a more appropriated exception to handle a null file? Can you please check? If not, I don't see as a problem we create a new exception. 😉

Reply:
Hi @natocesarrego, I was trying to maintain the same behavior as Documents and Media ([here](https://github.com/liferay/liferay-portal/blob/2d3d77fc7c6e0c407c1c633b8d1a12388720992a/modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/DLValidatorImpl.java#L197)) and I didn’t find another exception to handle a null file. Do you think we can create next to the others `(package com.liferay.document.library.kernel.exception)`?